docs/sphinx/developer/arch/supported/riscv.rst

```diff
@@ -48,7 +71,7 @@ Once the addresses have been generated, they should be supplied in a vector to t
 Pseudoinstructions
 ******************
 
-Similar to AArch64 instruction aliases, RISCV has many pseudoinstructions. These are usually specific instances of a more general instruction. They will have the same opcode but a different mnemonic. Capstone will disassemble these instructions giving a valid opcode but will often miss vital operands that need to be inferred from the instruction mnemonic. This can be quite dangerous as the instruction may pass through the pipeline completely unhindered, but will give an incorrect result. These sorts of errors are very hard to track down.
+Similar to AArch64 instruction aliases, RISC-V has many pseudoinstructions. These are usually specific instances of a more general instruction. They will have the same opcode but a different mnemonic. Capstone will disassemble these instructions giving a valid opcode but will often miss vital operands that need to be inferred from the instruction mnemonic. This can be quite dangerous as the instruction may pass through the pipeline completely unhindered, but will give an incorrect result. These sorts of errors are very hard to track down.
```

Comment:
May be better as `pseudo-instructions`

Reply:
It is written pseudoinstructions (no hyphen) in the official documentation so I have used that throughout